helm/values.yaml

```diff
@@ -168,13 +168,13 @@ smtp:
 notifiers:
   smtp:
     enabled: true
-    host: ${email.host}
-    subject: ${email.subject}
-    port: ${email.port}
-    from: ${email.from}
-    username: ${email.username}
-    password: ${email.password}
-    # starttlsEnabled: false
+    host: ${smtp.host}
+    port: ${smtp.port}
+    from: ${smtp.from}
+    subject: ${smtp.subject}
+    username: ${smtp.username}
+    password: ${smtp.password}
+    # starttlsEnabled: ${starttls.properties.enable}`
```

Comment:
is it a new value? If so, should we document it and let it start by smtp like the others
(smtp.starttls.properties...)

Reply:
Did we introduce a breaking with this change?

Reply:
@leleueri I updated that line. The value was there before but using it by default for "notifiers.smtp" is a new thing.

Reply:
@sebdevaux the value came from "${email.host}" before but I couldn't find "email.host" anywhere in the values. Did I missed something?